### PR TITLE
Feat: Adding inline-css-variables-in-svg

### PR DESCRIPTION
**Pull request by JulesGuesnon**
# What it does

This PR adds `inline-css-variables-in-svg` parser. It allows to replace all the `stroke` and `fill` values by a Css variable if a matching color token exists

# Additional informations
I need some feedbacks on a few points:
1. Currently the `cssVariablesFormat` option has no default value, should It?
2. To make the parser test works, it required to update `seeds.json`, but now some of the tests are broken (mostly snapshots). I only need a confirmation that it's the correct way of doing it before I fix all the tests.
3. I made a `README.md` based on the other parsers, feel free to tell me if there's lacking informations

### CODE REVIEW

File: tests/fixtures/seeds.json [hunk 1]
```diff
@@ -862,6 +862,46 @@
     "repositoryId": "927e26f6-da29-40d6-b9de-530bdcddf1ed",
     "description": "a description"
   },
+  {
```
Comment (AMoreaux) [inline tests/fixtures/seeds.json:865]: I will update seed to avoid manual addition.

**Review by AMoreaux — CHANGES_REQUESTED** [review]
Thanks for the quick PR 😁 Just a few change requests.

File: parsers/inline-css-variables-in-svg/inline-css-variables-in-svg.parser.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+    return result;
+  })(),
+
+  '#': svgNode['#']?.map(nodes => {
+    const ret: { [k: string]: SvgNode } = {};
+
+    for (const key in nodes) {
+      if (!Object.prototype.hasOwnProperty.call(nodes, key)) continue;
```
[the remark this answers is not in the document]
Comment (JulesGuesnon) [reply parsers/inline-css-variables-in-svg/inline-css-variables-in-svg.parser.ts:78]: Well, tbh it's the vscode `forin` snippet that includes it, and after some search, it looks it's for handling weird cases so I guess I'll remove it

File: tests/fixtures/seeds.json (hunk 1 above)
Comment (JulesGuesnon) [reply tests/fixtures/seeds.json:865]: Feel free to ping me when seed is updated!

Comment (AMoreaux) [reply tests/fixtures/seeds.json:865]: It's merged 👌

Comment (JulesGuesnon) [reply tests/fixtures/seeds.json]: Well, actually I still had tu update it as the color wasn't matching the svg otherwise 😅 Tell me if it's an issue

Comment (AMoreaux) [reply tests/fixtures/seeds.json]: I change seeds to use `AlmostBlack` explicitly in the stroke property of the vector `activity`

**Comment by JulesGuesnon** [comment]
@AMoreaux I think everything is good for review

**Review by AMoreaux — APPROVED** [review]
Nice 👌

**Pull request merged**